metrics/heartbeat_test.go

```diff
@@ -186,14 +186,12 @@ func TestHeartbeatRunSuccess(t *testing.T) {
 
 func mustMakeTipset(t *testing.T, height types.Uint64) types.TipSet {
 	ts, err := types.NewTipSet(&types.Block{
-		Miner:           address.NewForTestGetter()(),
-		Ticket:          nil,
-		Parents:         types.TipSetKey{},
-		ParentWeight:    0,
-		Height:          height,
-		Nonce:           0,
-		Messages:        nil,
-		MessageReceipts: nil,
+		Miner:        address.NewForTestGetter()(),
+		Ticket:       nil,
+		Parents:      types.TipSetKey{},
+		ParentWeight: 0,
+		Height:       height,
+		Nonce:        0,
```

Comment:
Why aren't you setting Messages and MessageReceipts here to the empty(Messages|Receipts)Cid as above?